content/doc/tutorials/tutorial-for-installing-jenkins-on-AWS.adoc

```diff
@@ -22,213 +20,218 @@ need to be augmented on the instance.
 
 == Prerequisites
 
-1. An AWS account, if you don't have one, you can register link:https://portal.aws.amazon.com/billing/signup#/start[here]
-2. An Amazon EC2 key pair, if you don't have one, see the <<Create a key pair using Amazon EC2>> section.
+1. An *AWS account*, if you don't have one, you can register link:https://portal.aws.amazon.com/billing/signup#/start[here].
+2. An Amazon EC2 key pair, if you don't have one, see the <<Creating a key pair>> section.
 
-== Create a key pair
+== Creating a key pair
+
+Creating a key pair will help ensure that the correct form of authentication is used when installing Jenkins.
 
 To create your key pair:
 
-1. Open the Amazon EC2 console at https://console.aws.amazon.com/ec2/ and sign in.
+. Open the Amazon EC2 console at https://console.aws.amazon.com/ec2/ and sign in.
 
-2. In the navigation pane, under **NETWORK & SECURITY**, choose **Key Pairs**.
+. In the navigation pane, under *NETWORK & SECURITY*, choose *Key Pairs*.
 
-3. Select **Create key pair**.
+. Select **Create key pair**.
 
-4. For **Name**, enter a descriptive name for the key pair. Amazon EC2 associates the public key with the name that you specify as the key name. A key name can include up to 255 ASCII characters. It can’t include leading or trailing spaces.
+. For *Name*, enter a descriptive name for the key pair.
+Amazon EC2 associates the public key with the name that you specify as the *key name*. 
+A key name can include up to 255 ASCII characters.
+It cannot include leading or trailing spaces.
 
-5. For **File format**, choose the format in which to save the private key. To save the private key in a format that can be used with OpenSSH, choose **pem**. To save the private key in a format that can be used with PuTTY, choose **ppk**.
+. For *File format*, choose the format in which to save the private key.
+* To save the private key in a format that can be used with OpenSSH, choose *pem*.
+* To save the private key in a format that can be used with PuTTY, choose *ppk*.
 
-6. Select **Create key pair**.
+. Select *Create key pair*.
 
-7. The private key file is automatically downloaded by your browser. The base file name is the name you specified as the name of your key pair, and the file name extension is determined by the file format you chose. Save the private key file in a safe place.
+. The private key file is automatically downloaded by your browser.
+The base file name is the name you specified as the name of your key pair, and the file name extension is determined by the file format you chose.
+Save the private key file in a safe place.
 +
-NOTE: This is the only chance for you to save the private key file.
+IMPORTANT: This is the only chance for you to save the private key file.
++
+. If you will use an SSH client on a macOS or Linux computer to connect to your Linux instance, use the following command to set the permissions of your private key file so that only you can read it.
```

Comment:
```suggestion
. If you use an SSH client on a macOS or Linux computer to connect to your Linux instance, run the following command to set the permissions of your private key file so that only you can read it.
```